Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreeTransaction.h

```diff
@@ -330,6 +334,8 @@ class RemoteLayerTreeTransaction {
     HashSet<RefPtr<PlatformCALayerRemote>> m_changedLayers; // Only used in the Web process.
     LayerPropertiesMap m_changedLayerProperties; // Only used in the UI process.
 
+    Markable<WebCore::FrameIdentifier> m_remoteFrameIdentifier;
```

Comment:
It's not super clear what this is. Is it the identifier for the rootmost frame in the process that's generating this transaction? We need a name for that concept (rootmost frame?)